docs/sophia.md

```diff
@@ -1,10 +1,14 @@
+<!--  IMPORTANT: This file shall temporarily remain as an internal reference for the transformation of the doc into the format favored by readthedocs and will be removed soon. For the actual documentation, see https://aeternity-sophia.readthedocs.io/-->
+
 <!-- IMPORTANT: REMEMBER TO UPDATE THE TABLE OF CONTENTS AFTER YOUR EDIT -->
 
 **Table of Contents**
 
+- [-](#-)
 - [Language Features](#language-features)
     - [Contracts](#contracts)
         - [Calling other contracts](#calling-other-contracts)
+        - [Protected contract calls](#protected-contract-calls)
```

Comment:
No... Again, this is not right. This is the Lima branch, this isn't implemented on Lima.

Reply:
this file was for internal reference only anyway.